notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiWithdrawalIntegrationTest.kt

```diff
@@ -56,10 +56,11 @@ class BtcMultiWithdrawalIntegrationTest {
             }
         val someEnvironment = environments.first()
         integrationHelper.generateBtcInitialBlocks()
-        integrationHelper.genChangeBtcAddress(someEnvironment.btcWithdrawalConfig.bitcoin.walletPath)
+        //TODO don't forget to add chain address to transfers wallet
```

Comment:
Is it relevant?

Reply:
This is very relevant. Will be fixed in next pr.